packages/react-components/react-table/src/stories/Table/SingleSelect.stories.tsx

```diff
@@ -129,7 +129,7 @@ export const SingleSelect = () => {
     <Table ref={ref}>
       <TableHeader>
         <TableRow>
-          <TableSelectionCell type="radio" />
+          <TableSelectionCell type="radio" hidden />
```

Comment:
Is there ever a case where you'd want to mix and match radios and checkboxes in a table? It seems like that would be a very confusing selection model. If not, you might want to have a `selectionMode="single"` (or `"radio"` ) on the table itself, instead of relying on each TableSelectionCell having the prop match.

Reply:
That is a legitmate concern of primitives  👍 We absolutely need a higher level component that adds this kind of consistency. However when you get consistency/convenienve you sacrifice flexibility. The intention of primitives is to be flexible, the future `DataGrid` component will provide convenience/consistency